src/Solnet.Programs/Models/Stake/State.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Solnet.Wallet;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Solnet.Programs.Models.Stake
+{
+    public class State
+    {
+        public struct Authorized
+        {
+            public PublicKey staker { get; set; }
+            public PublicKey withdrawer { get; set; }
+        }
+        public struct Delegation
+        {
+            public PublicKey voter_pubkey { get; set; }
```

Comment:
I think this will be my last comment, just regarding use of snake_case instead of PascalCase, again to maintain the code style all across the codebase.

Reply:
I think I got the spots where I did this.